pkg/cmd/util/cmd.go

```diff
@@ -57,25 +57,23 @@ func ResolveResource(defaultResource, resourceString string, mapper meta.RESTMap
 	case 1:
 		name = parts[0]
 	case 2:
-		partialResource := unversioned.GroupVersionResource{Resource: strings.ToLower(parts[0])}
-		gvrs, err := mapper.ResourcesFor(partialResource)
-		if err != nil {
-			return "", "", err
-		}
-		if len(gvrs) == 0 {
-			return gvrs[0].Resource, parts[1], nil
-		}
+		name = parts[1]
 
-		groupResource := gvrs[0].GroupResource()
-		for _, gvr := range gvrs[1:] {
-			if groupResource != gvr.GroupResource() {
-				return "", "", &meta.AmbiguousResourceError{PartialResource: partialResource, MatchingResources: gvrs}
-			}
+		// Allow specifying the group the same way kubectl does
```

Comment:
It would be ideal if we would re-use the code from there for all the parsing we do here.